zipline/utils/cli.py

```diff
@@ -99,9 +100,12 @@ def parse_args(argv, ipython_mode=False):
     parser.add_argument('--start', '-s')
     parser.add_argument('--end', '-e')
     parser.add_argument('--capital_base')
-    parser.add_argument('--source', choices=('yahoo',))
+    parser.add_argument('--source', '-d')
```

Comment:
Why is `choices` no longer specified here?

Also, most of these single-letter entries are non-obvious enough that I suspect we'd be better off not having them at all.